WasatchNET/Spectrometer.cs

```diff
@@ -1976,7 +1976,8 @@ string generateUniqueKey(UsbRegistry usbRegistry)
 
         public virtual bool isARM => featureIdentification.boardType == BOARD_TYPES.ARM;
         public bool isSiG => eeprom.model.ToLower().Contains("sig") || eeprom.detectorName.ToLower().Contains("imx");
-        
+        public virtual bool isInGaAs => featureIdentification.boardType == BOARD_TYPES.INGAAS_FX2;

```

Comment:
Okay for now, but we'll need to revisit when we release a SiG-NIR.  Long-term, we should probably just base this decision on the EEPROM's populated detector field.

Reply:
Starts with "G"?

Reply:
Could do.  This is how ENLIGHTEN does it: https://github.com/WasatchPhotonics/Wasatch.PY/blob/master/wasatch/SpectrometerSettings.py#L274